Detections/AzureFirewall/SeveralDenyActionsRegistered.yaml

```diff
@@ -0,0 +1,34 @@
+id: 
+name: Several deny actions registered
+description: |
+  'This Creates an incident when a source has more than 1 registered deny action in Azure Firewall.'
```

Comment:
I think it would be good to explain what type of security issue this is.  It would be unclear to users what the attack might be based on this.

Reply:
Agree. Edited.